src/java/org/apache/cassandra/service/accord/txn/TxnCondition.java

```diff
@@ -509,7 +581,6 @@ public long serializedSize(BooleanGroup condition, int version)
 
     public static final IVersionedSerializer<TxnCondition> serializer = new IVersionedSerializer<TxnCondition>()
     {
-        @SuppressWarnings("unchecked")
```

Comment:
nit: I still get a warning here without the suppression

Reply:
Huh, not sure why I touched that line. Added it back.